app/controllers/api/notes_controller.rb

```diff
@@ -294,11 +294,25 @@ def search
           raise OSM::APIBadUserInput, "Date #{params[:to]} is in a wrong format"
         end
 
-        @notes = @notes.where(:created_at => from..to)
+        @notes = params[:sort] == "created_at" ? @notes.where(:created_at => from..to) : @notes.where(:updated_at => from..to)
```

Comment:
This will change the behaviour of existing API calls - you should make `created_at` the default to preserve the existing behaviour when no sort is specified.

I'd also prefer it if this was written like the sort order code with a multiline if rather than using the ternary operator, which is much less readable.

Reply:
You're right. I didn't notice that one...